server/src/main/java/io/spine/server/rejection/RejectionHandlerMethod.java

```diff
@@ -152,11 +152,18 @@ Object doInvoke(Object target, Message rejectionMsg, RejectionContext context) {
          * A rejection handler method which receives a rejection message
          * as a single parameter.
          *
-         * <p>The signature of such a method is following:
+         * <p>The signature of such a method is as follows, if used in {@link RejectionSubscriber}:
          * <pre>
-         * [{@literal @}HandlerAnnotation]
+         * [{@literal @}Subscribe]
```

Comment:
Why do we need square brackets around `@Subscribe` here and in the other Javadocs below?

Reply:
@alexander-yevsyukov they were previously used to mark `[HandlerAnnotation]`.

Removed.